storage/spec/helpers.rb

```diff
@@ -0,0 +1,43 @@
+require "google/apis/cloudkms_v1"
+
+def create_kms_key project_id
+  kms_key_ring = ENV["GOOGLE_CLOUD_KMS_KEY_RING"]
```

Comment:
Make kms_key_ring and kms_key_name parameters. And get ENV variables in files_spec.rb instead.

Reply:
Done.